src/web/components/SharingPermission/SearchAndAddParticipants.tsx

```diff
@@ -51,63 +51,60 @@ export function SearchAndAddParticipants({
     return `${participantCount} Participants`;
   };
 
-  const handleSelectedParticipantChanged = (selectedItems: Set<number>) => {
+  const handleSelectedParticipantChanged = useCallback((selectedItems: Set<number>) => {
     setSelectedParticipants(selectedItems);
-    setOpenSearchResult(false);
-  };
+  }, []);
 
   return (
     <div className='search-and-add-participants'>
       <div className='add-participant-dialog-search-bar'>
         <ParticipantSearchBar
-          selectedParticipantIds={selectedParticipants}
           participants={getSearchableParticipants(availableParticipants)}
           onSelectedChange={handleSelectedParticipantChanged}
           participantTypes={participantTypes}
           open={openSearchResult}
           onToggleOpen={setOpenSearchResult}
         />
       </div>
-      {!openSearchResult && (
-        <div className='action-section'>
-          {selectedParticipants.size > 0 && (
-            <p>{getParticipantText(selectedParticipants.size)} added</p>
-          )}
-          <Dialog
-            title='Please review the following changes'
-            triggerButton={
-              <button
-                type='button'
-                className='primary-button add-participant-button'
-                disabled={!selectedParticipants.size}
-              >
-                Add Participants
-              </button>
-            }
-            open={openConfirmation}
-            onOpenChange={setOpenConfirmation}
-          >
-            Adding the following participants:
-            <ul className='dot-list'>
-              {selectedParticipantList.map((selectedParticipant) => (
-                <li key={selectedParticipant.id}>{selectedParticipant.name}</li>
-              ))}
-            </ul>
-            <div className='dialog-footer-section'>
-              <button type='button' className='primary-button' onClick={onHandleAddParticipants}>
-                Save
-              </button>
-              <button
-                type='button'
-                className='transparent-button'
-                onClick={() => setOpenConfirmation(false)}
-              >
-                Cancel
-              </button>
-            </div>
-          </Dialog>
-        </div>
-      )}
+
```

Comment:
I remove the openSearchResult here, as it was a bit annoying to dropdown closed everytime we select a participant